includes/elements/container.php

```diff
@@ -39,8 +39,6 @@ public function __construct( array $data = [], array $args = null ) {
 		parent::__construct( $data, $args );
 
 		$this->active_kit = Plugin::$instance->kits_manager->get_active_kit();
-		$this->logical_dimensions_inline_start = is_rtl() ? '{{RIGHT}}{{UNIT}}' : '{{LEFT}}{{UNIT}}';
-		$this->logical_dimensions_inline_end = is_rtl() ? '{{LEFT}}{{UNIT}}' : '{{RIGHT}}{{UNIT}}';
```

Comment:
This fails because the property is declared dynamically. I'd suggest declaring the property in the opening declaration:
```
private $active_kit;
private $logical_dimensions_inline_start;
private $logical_dimensions_inline_end;
```

Once that's done you should be able to keep the old code you had without this warning.

Reply:
I still smile everytime I see this LTR/RTL code `is_rtl() ? '{{RIGHT}}{{UNIT}}' : '{{LEFT}}{{UNIT}}';`
It's really smart

Reply:
Updated.

Unfortunately he 'is_rtl()' is causing a problem and will be removed from 3.18. The problem is that the user language used by the editor can be different from the page content. E.g. a user can use English on WordPress, but edit a page written in Hebrew.